pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -2411,6 +2439,15 @@ func parseIpspecNetworkXObject(ipspec *zconfig.Ipspec, config *types.NetworkXObj
 func parseIpspec(ipspec *zconfig.Ipspec,
 	config *types.NetworkInstanceConfig) error {
 
+	// Parse Subnet - mandatory argument.
+	if ipspec.GetSubnet() == "" {
```

Comment:
Do I get it right, that each network object would have to belong to a subnet now? What about upgrades to this version from an older version?

Reply:
Note that parseIpspec is called only when IP addressing is enabled (i.e. not for switch NI):
```
		if networkInstanceConfig.IpType != types.AddressTypeNone {
			err := parseIpspec(apiConfigEntry.Ip, &networkInstanceConfig)
...
```